xarray/tests/test_dask.py

```diff
@@ -1619,7 +1619,7 @@ def test_more_transforms_pass_lazy_array_equiv(map_da, map_ds):
         assert_equal(xr.broadcast(map_ds.cxy, map_ds.cxy)[0], map_ds.cxy)
         assert_equal(map_ds.map(lambda x: x), map_ds)
         assert_equal(map_ds.set_coords("a").reset_coords("a"), map_ds)
-        assert_equal(map_ds.update({"a": map_ds.a}), map_ds)
+        assert_equal(map_ds.assign({"a": map_ds.a}), map_ds)
```

Comment:
this seemed the easiest fix, but I could also do something like
```python
map_ds2 = map_ds.copy()
map_ds2.update({"a": map_ds.a})
```
but I'm not quite sure what the original line was supposed to check... `update` is in-place, so the check is the same as
```python
map_ds.update({"a": map_ds.a})
assert_equal(map_ds, map_ds)
```

Reply:
This looks good to me